packages/user/integration-tests/__fixtures__/user.ts

```diff
@@ -0,0 +1,16 @@
+import { SqlEntityManager } from "@mikro-orm/postgresql"
```

Comment:
thought(non-blocking): this approach of having one big seeder feels a bit off - think creating the data for the test within the test case is easier to wrap you your head around + you can use the service methods directly.

Reply:
Will update when introducing models today 😄